src/services/pickup_service.js

```diff
@@ -8,6 +8,20 @@ export default (easypostClient) =>
 
     static key = 'pickup';
 
+    /**
+     * Create a pickup.
+     * @param {*} params
```

Comment:
```suggestion
     * @param {object} params
```
nit: I am seeing `object` and `*` for the `params` parameter, we should stick to one of them.